### DISCUSSION TITLE
播放歌曲时，会不停往歌曲文件夹保存歌词

### DISCUSSION THREAD

**Discussion by kevinqingqinga** [zhongyang219/MusicPlayer2#666] [General]
播放歌曲时，会不停往歌曲文件夹保存歌词。这是要怎么设置才能解决这个问题？
![Snipaste_2024-02-17_21-59-12](https://github.com/zhongyang219/MusicPlayer2/assets/139779464/b02f86a9-1b89-4ccf-bd89-1eb1011caf5c)

**Comment by lrisora** [comment]
关掉 常规设置->“没有歌词时自动下载”
或者先设置 歌词设置->歌词文件夹 并将歌词保存位置设置为保存到歌词文件夹
注意“保存到歌词文件夹”这个设置有三处， 自动下载歌词/歌词下载窗口/歌词批量下载窗口

如果是V2.76那么升级到2.76.1，2.76有bug

**Comment by kevinqingqinga** [comment]
感谢回复！
________________________________
发件人: lrisora ***@***.***>
发送时间: 2024年2月17日 22:11
收件人: zhongyang219/MusicPlayer2 ***@***.***>
抄送: kevinqingqinga ***@***.***>; Author ***@***.***>
主题: Re: [zhongyang219/MusicPlayer2] 播放歌曲时，会不停往歌曲文件夹保存歌词 (Discussion #666)


关掉 常规设置->“没有歌词时自动下载”
或者先设置 歌词设置->歌词文件夹 并将歌词保存位置设置为保存到歌词文件夹
注意“保存到歌词文件夹”这个设置有三处， 自动下载歌词/歌词下载窗口/歌词批量下载窗口

―
